kubernetes/schema_volume_source.go

```diff
@@ -107,6 +107,12 @@ func commonVolumeSources() map[string]*schema.Schema {
 						Description: "Filesystem type to mount. Must be a filesystem type supported by the host operating system. Ex. \"ext4\", \"xfs\", \"ntfs\". Implicitly inferred to be \"ext4\" if unspecified.",
 						Optional:    true,
 					},
+					"kind": {
+						Type:        schema.TypeString,
+						Description: "The type for the data disk. Expected values: Shared, Dedicated, Managed. Defaults to Shared",
```

Comment:
Could add validation for the expected values?

Reply:
The API responds back with the specific values it's expecting if you give it the wrong one. If you leave it blank, the API makes it `kind: Shared` automatically, which fills in the computed value as `Shared`. Is that ok? Or should we still do some validation?

Reply:
I think that's OK then.